app/controllers/DownloadMetadataController.scala

```diff
@@ -22,14 +21,15 @@ class DownloadMetadataController @Inject() (
     val consignmentService: ConsignmentService,
     val customMetadataService: CustomMetadataService,
     val displayPropertiesService: DisplayPropertiesService,
-    val keycloakConfiguration: KeycloakConfiguration
+    val keycloakConfiguration: KeycloakConfiguration,
+    val applicationConfig: ApplicationConfig
 ) extends TokenSecurity {
 
   def downloadMetadataPage(consignmentId: UUID): Action[AnyContent] = standardTypeAction(consignmentId) { implicit request: Request[AnyContent] =>
     consignmentService
       .getConsignmentRef(consignmentId, request.token.bearerAccessToken)
       .map { ref =>
-        Ok(views.html.standard.downloadMetadata(consignmentId, ref, request.token.name))
+        Ok(views.html.standard.downloadMetadata(consignmentId, ref, request.token.name, applicationConfig.blockDraftMetadataUpload))
```

Comment:
Is that correct using the FAB for draft metadata? I thought the feature should be independent of whether draft metadata is available or not, ie even if a user uses the UI input method, it still needs to go to review?

I think the DTA Review feature needs its own FAB?

Reply:
I'm not sure about this. I thought the draft metadata upload and DTA Review would go live together. I'll confirm it with Emma. Thanks for raising this.

Reply:
Created a new feature access block for metadata review. Thanks